invenio_app_ils/documents/utils.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Document configuration callbacks."""
+
+
+def build_default_cover_urls(document):
+    """Build default ulrs for documents without cover."""
+    default_url = "/images/placeholder.png"
+    document["metadata"]["cover_urls"] = {
+        "small": "{0}".format(default_url),
+        "medium": "{0}".format(default_url),
+        "large": "{0}".format(default_url),
+    }
+    return document
+
+
+def build_literature_cover_urls(document):
```

Comment:
I think this method should be deleted, or partially moved to the CDS overlay.

Reply:
Yes, it will, this is my testing for syndetic and for the config override.